dist/gcds-css-shortcuts.css

```diff
@@ -19542,7 +19542,7 @@ table {
 }
 @font-face {
   font-family: "gcds-icons";
-  src: url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.ttf") format("truetype"), url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.woff") format("woff");
+  src: url("https://cdn.design-system.alpha.canada.ca/@gcds-core/fonts@1.0.0/icons/gcds-icons.ttf") format("truetype"), url("https://cdn.design-system.alpha.canada.ca/@gcds-core/fonts@1.0.0/icons/gcds-icons.woff") format("woff");
```

Comment:
Thanks for catching this! I would prefer to have this in a separate PR though if possible, do you mind removing it from this one?

Reply:
@radmorecameron would you mind merging the conflicts? That should be the last thing :)